internal/cmd/gocdk/serve.go

```diff
@@ -22,19 +22,25 @@ import (
 	"log"
 	"net"
 	"net/http"
+	"net/http/httputil"
 	"net/url"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
+	"sync/atomic"
 	"time"
 
+	"golang.org/x/sync/errgroup"
 	"golang.org/x/xerrors"
 )
 
 func serve(ctx context.Context, pctx *processContext, args []string) error {
 	f := newFlagSet(pctx, "serve")
-	address := f.String("address", "localhost:8080", "address to serve on")
-	biome := f.String("biome", "dev", "biome to apply and use configuration from")
+	opts := new(serveOptions)
+	f.StringVar(&opts.address, "address", "localhost:8080", "`host:port` to address to serve on")
+	f.StringVar(&opts.biome, "biome", "dev", "`name` of biome to apply and use configuration from")
+	f.IntVar(&opts.basePort, "base-port", 9090, "use `port` and port+1 for the built servers to listen on")
```

Comment:
`-address` specifies the port the user should connect to that has the reverse proxy and `-base-port` specifies the two ports that the proxied servers should run under. Open to better names for this.

Reply:
Just use constants for the latter? Or use the `--address` port +1 and +2?

Otherwise, I think you need to explain the architecture (that there is a reverse proxy, and 1-2 additional servers being proxied) in the flag. I don't think there's any flag name that will make any sense if the user doesn't know that.

Reply:
Used `-address` port +1 and +2, thanks for the idea! (I slightly modified for the case where the `-address` port is zero.)